examples/storm-perf/src/main/java/org/apache/storm/perf/utils/BasicMetricsCollector.java

```diff
@@ -192,10 +191,12 @@ void writeHeader(PrintWriter writer) {
             header.add(SPOUT_MAX_COMPLETE_LATENCY);
         }
 
-        writer.println("\n------------------------------------------------------------------------------------------------------------------");
+        writer.println(
+            "\n------------------------------------------------------------------------------------------------------------------");
         String str = Utils.join(header, ",");
         writer.println(str);
-        writer.println("------------------------------------------------------------------------------------------------------------------");
+        writer
+            .println("------------------------------------------------------------------------------------------------------------------");
```

Comment:
Would it be good to be consistent about this `writer.println()`? Here we have 
```
writer.println(
            "\n---...
```
and
```
 writer
            .println("---
```